canonical-web-and-design/multipass.run
# Add search to multipass docs

## QA

- Go to https://multipass-run-149.demos.haus/docs and search something like "application", which usually yields a lot of results
- Make sure the next button works and you can also search in the results page

## Issues
Fixes #148
